src/dune_config/compile_time.ml

```diff
@@ -0,0 +1,19 @@
+open Stdune
+
+let initialized = ref false
+let toggled = ref []
+
+let init ~names =
+  if !initialized then Code_error.raise "Compile_time.init: already initialized" [];
+  toggled := names;
+  initialized := true
+;;
+
+let set name =
```

Comment:
I think `make` would be a more appropriate name.

Reply:
Done!